presto-main/pom.xml

```diff
@@ -243,6 +244,12 @@
             <artifactId>asm-all</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.jgrapht</groupId>
+            <artifactId>jgrapht-core</artifactId>
+            <version>${dep.jgrapht.version}</version>
```

Comment:
Instead of putting the version here, put it in the dependency management section of the root pom.xml (the one in the root of the presto repo)